dev-lang/vala/vala38-0.38.8.recipe

```diff
@@ -0,0 +1,131 @@
+SUMMARY="An object-oriented programming language that uses the GObject system"
+DESCRIPTION="Vala is a new programming language that aims to bring modern \
+programming language features to GNOME developers without imposing any \
+additional runtime requirements and without using a different ABI \
+compared to applications and libraries written in C."
+HOMEPAGE="https://wiki.gnome.org/Projects/Vala"
+COPYRIGHT="2006-2018 Jürg Billeter
+	2006-2018 Raffaele Sandrini"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/vala/${portVersion%.*}/vala-$portVersion.tar.xz"
+CHECKSUM_SHA256="2fa746b51cd66e43577d1da06a80b708c2875cadaafee77e9700ea35cf23882c"
+SOURCE_DIR="vala-$portVersion"
+PATCHES="vala38-$portVersion.patchset"
```

Comment:
You should either:
* fix the `PROVIDES`, `PROVIDES_devel`, and `defineDebugInfoPackage` (by adding a "38" at the appropriate places), and add a `CONFLICTS`, or
* rename the recipe to `vala-0.38.8.recipe` (and the patch to `vala-0.38.8.patchset`, of course) and make this change:
```
-SOURCE_DIR="vala-$portVersion"
-PATCHES="vala38-$portVersion.patchset"
+PATCHES="vala-$portVersion.patchset"
```

It seems vala-0.36.12 is almost OK and maybe we should merge it before this one.

Reply:
Right there, probably be something for this weekend, don't know if I get around to it in the next days :)